storage-proofs/porep/src/stacked/vanilla/cores.rs

```diff
@@ -62,44 +62,50 @@ fn get_thread_id() -> ThreadId {
 
 pub struct Cleanup {
     tid: ThreadId,
-    prior_state: Option<hwloc::Bitmap>,
+    prior_state: Option<hwloc2::Bitmap>,
 }
 
 impl Drop for Cleanup {
     fn drop(&mut self) {
         if let Some(prior) = self.prior_state.take() {
-            let child_topo = &TOPOLOGY;
-            let mut locked_topo = child_topo.lock().expect("poisded lock");
-            let _ = locked_topo.set_cpubind_for_thread(self.tid, prior, CPUBIND_THREAD);
+            if let Some(ref mut locked_topo) = &mut *TOPOLOGY.lock().expect("poisded lock") {
```

Comment:
Let's fix that typo while we're making changes here.

Reply:
I approved on the assumption this is intended to just make mechanical changes to facilitate library version upgrade. CI is failing, though, so further tweaking may be required.

Reply:
all CI is failing atm, will work on that after that is fixed in #1358